_posts/0000-01-02-Tank44-22.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Education is the learning of facts, but the training of the mind to think." - Albert Einstein